spring-cloud-gcp-docs/src/main/asciidoc/spring-stream.adoc

```diff
@@ -29,25 +29,45 @@ A https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-
 
 This binder binds producers to Google Cloud Pub/Sub topics and consumers to subscriptions.
 
-NOTE: Partitioning and consumer groups are not currently supported by this binder.
+NOTE: Partitioning is currently not supported by this binder.
 
 === Configuration
 
-You can configure the Spring Cloud Stream Binder for Google Cloud Pub/Sub to automatically generate the underlying resources, like the Google Cloud Pub/Sub subscriptions for the consumers.
-For that, you can use the `spring.cloud.stream.gcp.pubsub.bindings.[CHANNEL-NAME].consumer.auto-create-resources` property, which is turned ON by default.
+You can configure the Spring Cloud Stream Binder for Google Cloud Pub/Sub to automatically generate the underlying resources, like the Google Cloud Pub/Sub topics and subscriptions for producers and consumers.
+For that, you can use the `spring.cloud.stream.gcp.pubsub.bindings.<channelName>.<consumer|producer>.auto-create-resources` property, which is turned ON by default.
 
-If automatic resource creation is turned ON and the subscription and the topic do not exist for a consumer, a subscription and a topic will be created with the same name.
-For example, for the following configuration, a topic and a subscription called `myConsumer` would be created.
+Starting with version 1.1, these and other binder properties can be configured globally for all the bindings, e.g. `spring.cloud.stream.gcp.pubsub.default.consumer.auto-create-resources`.
```

Comment:
The Docs are great, although this option makes me think that with an anonymous group we just should ignore it and always create a subscription.

Reply:
I think you're fight, because in that scenario, there is no way the user can have the subscription already created.

Reply:
Created #1180